docs/2.0/docs/accountfactory/architecture/security-controls.md

```diff
@@ -0,0 +1,231 @@
+
+By default, the only repository/group required to interact with infrastructure using Pipelines in Gruntwork Account Factory is the `infrastructure-live-root` repository/group. This contains infrastructure code for `management`, `logs`, `security`, and `shared` accounts. Access should be limited to a small, trusted group responsible for defining critical infrastructure, similar to the role of the `root` user in Unix systems.
+
+## Platform-Specific Access Controls
+
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+
+<Tabs groupId="platform">
+<TabItem value="github" label="GitHub" default>
+
+- The AWS IAM role assumed via OIDC when pull requests are opened or updated has a trust policy that restricts access to the repository itself and provides read-only permissions
+- The AWS IAM role assumed via OIDC when pull requests are merged into the `main` branch has a trust policy limiting access to the repository's `main` branch and granting write permissions
+- Branch protection rules can be configured to require reviews and status checks
+- GitHub App or machine user authentication options available
+
+</TabItem>
+<TabItem value="gitlab" label="GitLab">
+
+- The AWS IAM role assumed via OIDC when merge requests are opened or updated has a trust policy that restricts access to the group itself and provides read-only permissions
+- The AWS IAM role assumed via OIDC when merge requests are merged into the `main` branch has a trust policy limiting access to the group's `main` branch and granting write permissions
+- Protected branches can be configured to require approvals and pipeline success
+- Machine user authentication required with group-level access configuration
+
+</TabItem>
+</Tabs>
+
+## Infrastructure access control
+
+An optional `infrastructure-live-access-control` repository/group can manage access control for infrastructure provisioned in AWS accounts. Using this is a best practice for centralized and auditable access management.
+
+- Access to the `main` branch should be restricted to a small, trusted group managing infrastructure access
+- The same least privilege principles apply: roles assumed for pull/merge requests have read-only permissions, while roles for merged changes have write permissions
+
+Unlike the infrastructure-live-root repository, this repository focuses on managing access control rather than defining infrastructure. You might grant write access to a broader group for managing access while maintaining tight control over the main branch. Encourage collaboration between platform teams and application engineers to review and refine access control continuously.
+
+## Token Strategy
+
+<Tabs groupId="platform">
+<TabItem value="github" label="GitHub" default>
+
+### GitHub App Installation Strategy (Recommended)
+
+No tokens are required when using the GitHub App.
+
+### Machine Users Installation Strategy
```

Comment:
It's not clear to me if machine users are an alternative or supplement to the github app. Maybe clarify by saying something like "You have the following options when configuring GitHub tokens."